lib/pf/services/manager/keepalived.pm

```diff
@@ -58,7 +58,8 @@ sub generateConfig {
 
     my %tags;
     $tags{'template'} = "$conf_dir/keepalived.conf";
-    $tags{'emailaddr'} = $Config{'alerting'}{'emailaddr'};
+    # split email addresses then rejoin them with line feed
+    $tags{'emailaddr'} = join("\n", split( /\s*,\s*/, $Config{'alerting'}{'emailaddr'}));
```

Comment:
Guys,

I need your help to have a well indented `keepalived` generated config. With current code, I get:
```
global_defs {
   notification_email {
     foo@example.com
bar@example.com
}
```

Reply:
Try something like this
```perl
    $tags{'emailaddr'} = join("\n     ", split( /\s*,\s*/, $Config{'alerting'}{'emailaddr'}));
```

Reply:
Since this is very specific to keep alive at the moment just create a constant in lib/pf/services/manager/keepalived.pm.

However if you feel that they should be global constant put it in pf/constants.pm